index.html

```diff
@@ -13107,13 +13107,27 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
         <div class="property" id="aria-controls">
           <pdef>aria-controls</pdef>
           <div class="property-description">
-            <p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the current element. See related <pref>aria-owns</pref>.</p>
-            <p>For example:</p>
+            <p>
+              <a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the focused element or composite widget. See related <pref>aria-details</pref> and
+              <pref>aria-owns</pref>.
+            </p>
+            <p>
+              The <code>aria-controls</code> property is for referencing elements which are modified through user interaction of the current element or composite widget the user is interacting with.
+              The presence of <code>aria-controls</code> enables <a>assistive technologies</a> to make users aware of the current element's association with the controlled elements, or that an
+              association exists for instances where the controlled element was previously in the hidden state, but through user interaction of the controlling element, the associated element has
+              become accessible.
```

Comment:
adding in @jnurthen's suggestion. tried simplifying the first sentence further.
```suggestion
              The <code>aria-controls</code> property is for referencing elements that are modified by the user interacting with the currently focused element or composite widget.
              The presence of <code>aria-controls</code> enables <a>assistive technologies</a> to make users aware of the current element’s association with the controlled element or elements. It can also inform them that an
              association exists for instances where the controlled element was previously in the hidden state and through user interaction of the controlling element, the associated element has
              become accessible.
```